Models/Artefact.cs

```diff
@@ -13,15 +13,14 @@ public class Artefact
         [JsonConverter(typeof(StringEnumConverter))]
         [JsonRequired]
         public Visibility Visibility { get; set; }
-
         [JsonRequired]
         public string Description { get; set; }
         public System.DateTime CreatedAt { get; set; }
+
         public IEnumerable<ArtefactCategory> CategoryJoin { get; set; }
-        // [InverseProperty("Artefact")]
         public IEnumerable<ArtefactDocument> Images { get; set; }
-        // [InverseProperty("Artefact")]
-        // public ICollection<ArtefactDocument> Documents { get; set; }
+        public IEnumerable<ArtefactComment> Comments { get; set; }

```

Comment:
May be one of those 'misunderstanding dotnet' moments but does this mean we'll get all comments on an artefact every time we load that artefact from the DB? 
That's bad.

Reply:
No it doesn't - to load all comments, would need to do a -
`_context.Artefacts.Include(artefact => artefact.Comments)`

https://www.learnentityframeworkcore.com/lazy-loading

I don't think we have lazy-loading enabled at the moment, but it's just a property in `ApplicationDbContext`

Reply:
Okay, misunderstanding it is.